tests/integration/conftest.py

```diff
@@ -0,0 +1,119 @@
+"""Pytest configuration and shared fixtures for integration tests.
+
+For shared and importable helpers, use helpers.py instead.
+"""
+
+import functools
+import logging
+import os
+import shutil
+from collections import defaultdict
+from datetime import datetime
+from pathlib import Path
+from typing import Union
+
+import pytest
+import yaml
+from helpers import PackedCharm
+from pytest_jubilant import pack
```

Comment:
```suggestion
from pytest_jubilant import pack, get_resources
```

instead of your custom `get_charm_resources`

Reply:
done!